thejamesgore/Foxtons-Dashboard
# feat: Add tests for root path in Express server

## Description
🚀 This pull request adds tests to verify the functionality of the Express server.

## Changes Made
👉 Added tests using Jest and Supertest to test the root path of the Express server. The tests verify that the server responds to the GET method, returns JSON data, and returns the expected message.

## Reason for Changes
🔍 Adding tests to the codebase helps ensure that the application works as expected and prevents regressions when making changes in the future. By testing the root path of the Express server, we can be confident that this critical piece of functionality is working as expected.

## How to Test
To test these changes, you can follow these steps:
1. 🧑‍💻 Clone this branch to your local machine
2. 📦 Install the necessary dependencies with `npm install`
3. 🏃‍♀️ Run the tests with `npm test`
4. ✅ Verify that the tests pass and that the application behaves as expected

## Checklist

## Related Issues
None.

## Other Notes
None.
